Minutes.java

```diff
@@ -1,13 +1,27 @@
 class Minutes extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * Minutes.java lets you enter a number of minutes, and that will 
+  * calculate the number of days, hours and minutes that represents.
+  * @author: E. Fung
   */
   
   public void run() {
-    
-    // start coding here
+    // initializing variables
+    int intMins;
+    int intDays;
+    int intHours;
+
+    // input 
+    intMins = readInt("Please enter number of minutes: ");
+
+    // calculation
+    intDays = intMins / 1440;
+    intHours = (intMins % 1440) / 60;
+    intMins = (intMins % 1440) % 60;
```

Comment:
used three lines and three variables to do calculations instead of making two more integers to represent the excess hours and minutes